docs/feature-roles.asciidoc

```diff
@@ -91,10 +92,6 @@ that has the following privileges:
 |Write events into {es}
 |====
 
-. If <<configuration-rum,real user monitoring>> is enabled, additional privileges are required to read source maps.
-See {kibana-ref}/rum-sourcemap-api.html[RUM source map API] for more details.
-Assign these extra privileges to the *general writer role*.
-
```

Comment:
Maybe keep this section too and refer it to the new `privileges-rum-source-mapping`? While not directly related, reading source maps is related to writing RUM events as expected.